buildrunner/sshagent/__init__.py

```diff
@@ -42,30 +43,19 @@ def load_ssh_key_from_file(key_file, passwd):
     """
     Load the given keys into paramiko PKey objects.
     """
-    try:
-        return RSAKey.from_private_key_file(key_file, passwd)
-    except PasswordRequiredException as pwdreqe:
-        raise BuildRunnerConfigurationError(
-            f"Key at {key_file} requires a password"
-        ) from pwdreqe
-    except SSHException:
+    key_types = [RSAKey, ECDSAKey, Ed25519Key, DSSKey]
```

Comment:
Shouldn't this be a constant at the top level of the file?

Reply:
That might be better, yeah